participants/Filvr.md

```diff
@@ -0,0 +1,80 @@
+## (0) Registration Submission, Project Name, and Team Slack Handles
+* Registration Submission ID: **670225568**
+* Project Name: **FilVR**
+* Team Slack Handles: **filvr.slack.com**
+
+
+## (1) Brief Project Description 
+**FilVR is a App build by IPSB team. The application is mainly used to manage VR video, provide users with video upload, download, search, view and so on.**
+
+
+## (2) Link to Project UI 
+* Project URL: **http://filvr.ipsb.io**
+
+
+## (3) What does your application/UI do? 
+* **User login / register **
+* **Show VR video Datas**
+* **Upload VR video**
+* **Retrieve VR video**
+* **Edit video data info**
+
+
+## (4) If your project is using a curated dataset, which one are you using?
+**N/A**
+
+
+## (5) If your project is not using a curated dataset, please tell us a little bit more about your data by answering the questions below.
+- **VR video data**
+- **Video format: MP4,AVI,FLV...**

```

Comment:
The Condensed Movie archive may be another choice to rapidly increase the storage on your application.  https://www.robots.ox.ac.uk/~vgg/research/condensed-movies/